ts-server/.editorconfig

```diff
@@ -0,0 +1,21 @@
+; top-most EditorConfig file
+root = true
+
+[*]
+indent_style = tab
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[package.json]
+indent_size = 2
+
+[*.yml]
+indent_style = space
+indent_size = 2
+
+[*.md]
+indent_style = space
```

Comment:
Using tabs to indent pre-formatted code in markdown makes the spacing too extreme when rendered on GitHub, so we revert back to spaces here.